parser/src/ast/structure.rs

```diff
@@ -0,0 +1,16 @@
+use crate::ast::Expr;
+use dbg_pls::DebugPls;
+
+/// A constructor call.
+///
+/// This differs from a function call in that it creates a new instance of a
+/// type, rather than perform a specific task. Constructors are always named
+/// and are always called with parentheses.
+#[derive(Debug, Clone, PartialEq, DebugPls)]
+pub struct Construct<'a> {
```

Comment:
`ConstructorCall` looks more accurate, `Construct` should rather be for Constructor definitions

Reply:
*Construct* refers to the action of creating an instance, while *Constructor* refers to the actual definition that is called when constructing an instance. While `ConstructorCall` would work, I think that `Construct` is more in line with the other *short* names we have in the AST.